editor/plugins/plugin_config_dialog.cpp

```diff
@@ -109,6 +109,8 @@ void PluginConfigDialog::_on_canceled() {
 void PluginConfigDialog::_on_required_text_changed() {
 	if (name_edit->get_text().is_empty()) {
 		validation_panel->set_message(MSG_ID_PLUGIN, TTR("Plugin name cannot be blank."), EditorValidationPanel::MSG_ERROR);
+	} else if (name_edit->get_text().contains("/") || name_edit->get_text().contains("\\")) {
+		validation_panel->set_message(MSG_ID_PLUGIN, TTR("Plugin name is invalid."), EditorValidationPanel::MSG_ERROR);
```

Comment:
You could state the reason why it's invalid (i.e. it contains slashes), since it's the only case.